api/src/main/java/io/kafbat/ui/serdes/builtin/sr/SchemaRegistrySerde.java

```diff
@@ -166,6 +169,11 @@ private static SchemaRegistryClient createSchemaRegistryClient(List<String> urls
           keyStorePassword);
     }
 
+    if (!verifySsl) { // TODO block above
```

Comment:
Not sure, what this comment mean. Could you please elaborate?

Reply:
It's a test change :) ignoring ssl doesn't work for the SR serde, looking for a fix